src/cli/index.js

```diff
@@ -349,10 +349,10 @@ export function main({
     return errorReportLoc;
   }
 
-  const exit = exitCode => {
+  function exit(exitCode) {
```

Comment:
Why convert to "normal" functions from arrow functions?

Reply:
Because otherwise `exit` is undefined at lines 178 & 195, since the assignment hasn't been executed yet.

Reply:
Ah, let's move this above then? Or add a comment if moving up is not possible?